src/templates/header/partials/nav-icon.hbs

```diff
@@ -0,0 +1,4 @@
+<svg class="c-nav-icon" xmlns="http://www.w3.org/2000/svg" width="35" height="35" viewBox="0 0 35 35">
```

Comment:
Does this need SVG-squishing?

Reply:
It has already been minified so it should be fine.

Having the SVG embedded in this way does raise a question around how we should be embedding SVG assets in these templates — we have the asset helper for C# code to handle embedding assets from the file system, should we consider something similar for these sharable templates? 

I don't think it's something we need to worry about too much at the moment, but I think it's something we need to keep in mind.

Reply:
The reason I asked is because the gulp one removes new lines, so I didn't expect the file to be 4 lines long.

Reply:
I'll remove the new lines, there were added before everything was pulled int partials so it was easy to scan the (large) document and see that it was an SVG.

Reply:
I mean, sure it's a micro-optimisation, but having them just made me think it hadn't been smushed 😄